File: DataAccessLayer/Models/Driver.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataAccessLayer.Models
{
    public class Driver
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string PhoneNumber { get; set; }
        public string DriversLicenceNumber { get; set; }

        public override string ToString()
        {
            return $"{Id} - {FirstName} {LastName}";
        }

        public string TestPrint()
        {
            return $"{Id}\n{FirstName}\n{LastName}\n{PhoneNumber}\n{DriversLicenceNumber}";
        }
    }
}

File: bulk-import-module/src/hr/mucnjakf/models/Driver.java
package hr.mucnjakf.models;

public class Driver {

    private String firstName;
    private String lastName;
    private String phoneNumber;
    private String driversLicenceNumber;

    public Driver(String firstName, String lastName, String phoneNumber, String driversLicenceNumber) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.phoneNumber = phoneNumber;
        this.driversLicenceNumber = driversLicenceNumber;
    }

    public String getFirstName() {
        return firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public String getPhoneNumber() {
        return phoneNumber;
    }

    public String getDriversLicenceNumber() {
        return driversLicenceNumber;
    }

    @Override
    public String toString() {
        return "Driver{" +
                "firstName='" + firstName + '\'' +
                ", lastName='" + lastName + '\'' +
                ", phoneNumber='" + phoneNumber + '\'' +
                ", driversLicenceNumber='" + driversLicenceNumber + '\'' +
                '}';
    }
}
